bokeh/core/property/color.py

```diff
@@ -66,14 +66,16 @@ def validate(self, value, detail=True):
 class Color(Either):
     """ Accept color values in a variety of ways.
 
-    For colors, because we support named colors and hex values prefaced
-    with a "#", when we are handed a string value, there is a little
-    interpretation: if the value is one of the 147 SVG named colors or
-    it starts with a "#", then it is interpreted as a value.
-
-    If a 3-tuple is provided, then it is treated as an RGB (0..255).
-    If a 4-tuple is provided, then it is treated as an RGBa (0..255), with
-    alpha as a float between 0 and 1.  (This follows the HTML5 Canvas API.)
+    * If a color is provided as a string, Bokeh determines whether this string
+      represents one of the named CSS colors (such as "red"), a CSS4 color
+      string (such as "rgb(0, 200, 0)"), or a hex value (such as "#00FF00").
+    * If a 3-tuple is provided, it is treated as an RGB value (between 0 and
+      255).
+    * If a 4-tuple is provided, it is treated as an RGBA value (between 0 and
+      255),
+      with alpha as a float between 0 and 1 (this follows the HTML5 Canvas API).
```

Comment:
weird line break here

Reply:
Also maybe be specific about RGB:
> If a 4-tuple is provided, it is treated as an RGBA value, with RGB values between 0 and 255, and  with alpha as a float between 0 and 1 (his follows the HTML5 Canvas API).

and similar above.

Reply:
>  (this follows the HTML5 Canvas API).

It depends on which part of the API. Alpha can be a byte as well. I wouldn't mention canvas at all, as we are not exposing raw canvas to the user, but abstracting over it and other rendering backends as well.

Reply:
"This follows the HTML5 Canvas API" was in the previous version of the docstring - I agree that we should best leave this out though. I'll commit an updated version soon!